lib/Assets.php

```diff
@@ -92,6 +92,39 @@ public function enqueue( $directory, $opts = [] ) : void {
 		}
 	}
 
+	/**
+	 * Enqueue our remote react app assets.
+	 *
+	 * @param string $base_url Base URL to our remote react app.
+	 */
+	public function enqueue_remote( $base_url ): void {
```

Comment:
Everything is great, but I noticed that `$cra_directory` is exclusively for getting the asset-manifest.json. So, the auto-appending kinda threw me off because I would assume you're looking for a asset-manifest.json like the other enqueue method, but I know this is specifically to support, so I might be making a contrived point. Because if things did change, then we can just talk and update. Just thinking out loud. Approved!

Reply:
Thanks for the feedback @tarikhamilton! 😄